src/App.js

```diff
@@ -0,0 +1,24 @@
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import Navbar from './components/navbar';
+import Books from './components/books';
+import Category from './components/category';
+import Form from './components/form';
+
+function App() {
+  return (
+    <Router>
+      <div>
+        <Navbar />
+        <div className="content">
+          <Routes>
+            <Route path="/" element={<Books />} />
+            <Route path="/category" element={<Category />} />
+            <Route path="/form" element={<Form />} />
```

Comment:
- According to the project requirements, there should be only 2 routes added in this component, `Book` and `Category`. I suggest you remove the `Form` route since it is already rendered in your `Book` component, please fix this 🙏